wikibase.py

```diff
@@ -42,13 +41,9 @@ def update_page(title, page, text, reason, ot,
     if text == ot:
         return None
 
-    if wiki_dry_run:
-        print("WIKI DRY RUN, skipping page.edit()")
-        return text
-    else:
-        update = page.edit(text, reason).get("edit", {"nochange": ""})
-        if "nochange" in update:
-            return None
+    update = page.edit(text, reason).get("edit", {"nochange": ""})
+    if "nochange" in update:
+        return None
```

Comment:
This will return None regardless if there was a change or not, because functions return None. I don't remember if this is important or not, but I think this should return "text" if the update was successful.

Reply:
Thanks! I missed that, and checked the file history to be sure everything else is reverted as well.